lisa/tools/mount.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import re
+from pathlib import PurePosixPath
+from typing import Any, cast
+
+from lisa.executable import Tool
+from lisa.operating_system import Posix
+from lisa.util import LisaException
+
+
+class Mount(Tool):
+    __UMOUNT_ERROR_PATTERN = re.compile(r".*mountpoint not found", re.MULTILINE)
+
+    @property
+    def command(self) -> str:
+        return self._command
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def mount(self, disk: str, point: str) -> None:
+        self._command = "mount"
```

Comment:
It doesn't need to save the self._command, because this value is only effective in this method. In the following code, it can call `self.node.execute`